topbin/dune

```diff
@@ -1,37 +1,44 @@
-(install
- (section bin)
- (package coq-core)
- (files (coqtop_bin.exe as coqtop)))
-
 (executable
  (name coqtop_bin)
  (public_name coqtop.opt)
  (package coq-core)
  (modules coqtop_bin)
  (libraries coq-core.toplevel findlib.dynload))
 
+(install
+ (section bin)
+ (package coq-core)
+ (files (coqtop_bin.exe as coqtop)))
+
 (executable
  (name coqtop_byte_bin)
  (public_name coqtop.byte)
  (package coq-core)
  (modules coqtop_byte_bin)
- (libraries compiler-libs.toplevel coq-core.toplevel findlib.top)
+ (libraries compiler-libs.toplevel coq-core.config.byte coq-core.toplevel coq-core.top_printers findlib.top)
```

Comment:
Not sure we want to unconditionnally link top_printers

Reply:
Why not? It is only for the bytecode toplevel.

A side effect of this change that I noticed was that `Drop` is now much faster.